examples/src/main/java/io/dapr/examples/pubsub/http/SubscriberController.java

```diff
@@ -26,7 +26,7 @@
    * @param cloudEvent The cloud event received.
    * @return A message containing the time.
    */
-  @Topic(name = "testingtopic", pubsubName = "messagebus")
+  @Topic(name = "testingtopic", pubsubName = "${myAppProperty:messagebus}")

```

Comment:
`myAppProperty` does not exist. Let's see if the auto-validation of the examples will pass.

Reply:
Yes - I did that intentionally. The idea is to demonstrate that while myAppProperty doesn't exist - it will use the default value in the expression, messagebus ; thus passing the validation :)

Reply:
Ah. OK. Can this be explained as comment in the code and text in README?

Reply:
Updated as requested.

Reply:
The comment in the java file caused a checkstyle error. I removed it and kept the README instructions. Thanks.